### ISSUE TITLE
How to get a count of modified documents via update

### ISSUE THREAD

**Issue by rzer** [dj-nitehawk/MongoDB.Entities#31]
For example we have a code:
```csharp
DB.Update<Profile>()
  .Match(a => a.Nick== nick && a.Money >= 10)
  .Modify(b => b.Inc(a => a.Money , -10))
  .Modify(b => b.CurrentDate(a => a.ModifiedOn))
  .Execute();
```

How can we understand was it applied or not? Can we get a  count of modified documents?

PS. Thanks for the library.

**Comment by dj-nitehawk** [comment]
if the operation completed without throwing an exception, that means the update was applied completely.

i never had the need to get a count of docs affected by an update operation so i omitted this functionality from the library.

if it's a high priority requirement for you, give me a couple of days to sort something out. i'm sure it can be added easily.

thanks!

**Comment by rzer** [comment]
Thanks!

I found that your library is the great wrapper for standart MongoDB driver. It's only one barrier for me now to use it in my project. This functionality will be very helpfull for me. I really appreciate your work!

**Comment by paolobriones** [comment]
@rzer me too, been using the MongoDB.Entities API for almost a year now; millions of transactions and it has been reliable and very stable!

**Comment by dj-nitehawk** [comment]
@rzer just released v11.8. all save/update/delete endpoints now return result where you can get the stats you wanted like in official driver.

@paolobriones thanks for the vote of confidence buddy!

**Comment by rzer** [comment]
Wow, great work!

**Comment by dj-nitehawk** [comment]
@rzer i saw that you've begun to add `DB.FindOneAndUpdate()` functionality. i was working on this myself and i've released v12.0 with `DB.UpdateAndGet()` which does the same thing.

**Comment by rzer** [comment]
Perfect! Switched to your  [version](https://rzer.ru/snapshots/snapshot_rvzEd4.png)
